modules/core/src/main/java/org/apache/ignite/internal/processors/job/GridJobProcessor.java

```diff
@@ -715,6 +717,21 @@ public void masterLeaveLocal(IgniteUuid sesId) {
                 job.onMasterNodeLeft();
     }
 
+    /** Searches for jobs that satisfy the condition.
```

Comment:
It seems comment should be placed on the new line.

Reply:
Sure. Done.